contrib/hawq-docker/centos6-docker/hawq-test/conf/slaves

```diff
@@ -0,0 +1 @@
+localhost
```

Comment:
This is the hdfs slaves file. If we start a hdfs cluster, this slaves should contains the 3 hostname , right ? Could you point me when do you update this slaves from "localhost" to "datanode1 datanode2 datanode3"  ? Maybe I missed it.  Thanks